[windows] add some ci tests for ETW parsing

derekwbrown opened the pull request:
### Describe how to test/QA your changes

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [x] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [x] Changed code has automated tests for its functionality.
- [x] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

derekwbrown committed:
[windows] add some ci tests for ETW parsing

derekwbrown committed:
lint

derekwbrown committed:
move different sites so they don't collide with other tests

brycekahle reviewed (approved):
We should consider changing CODEOWNERS for these windows-specific files

derekwbrown committed:
Convert transaction type to []byte when generating types.

make test code more reusable

derekwbrown committed:
Do better job of parsing output.
Add tests for paths on boundary of the fragment length
fix bug where ETW would return that the path was truncated always

derekwbrown committed:
gofmt

derekwbrown committed:
add validation of method

derekwbrown committed:
rearrange the support code to be more clear (review)

derekwbrown committed:
change variable name
